src/components/Modal/__test__/modal.spec.js

```diff
@@ -187,4 +188,22 @@ describe('<Modal/>', () => {
         component.unmount();
         expect(CounterManager.decrement).not.toHaveBeenCalled();
     });
+    it('should render the close button by default', () => {
+        const component = mount(
+            <Modal isOpen>
+                <p />
+            </Modal>,
+        );
+
+        expect(component.find(StyledCloseButton).exists()).toBe(true);
```

Comment:
I changed this because when you said:
```expect(component.find(StyledCloseButton).exists).toBeTruthy();```
exists is a method in the component.find(StyledCloseButton) object and then you are accessing to the method declaration that is truthy instead of calling it